apps/main/src/modules/layout/components/Header.tsx

```diff
@@ -0,0 +1,60 @@
+import { Flex, Toggle } from "@galacticcouncil/ui/components"
+import { useTheme } from "@galacticcouncil/ui/theme"
+import { getToken } from "@galacticcouncil/ui/utils"
+import { Link } from "@tanstack/react-router"
+
+const NAVIGTION = [
```

Comment:
I'd move it out from Header.tsx component. Let's have a utils file for things like those

Reply:
its just a mockup for now, can be done when implementing real navigation